test/test_coordgen.cpp

```diff
@@ -179,3 +179,28 @@ BOOST_AUTO_TEST_CASE(ClearWedgesTest)
     BOOST_REQUIRE_EQUAL(mol->getBonds().at(2)->hasStereochemistryDisplay, false);
     BOOST_REQUIRE_EQUAL(mol->getBonds().at(3)->hasStereochemistryDisplay, true);
 }
+
+
+BOOST_AUTO_TEST_CASE(DisableMetalZOBs)
+{
+    // Load a molecule with a single bond to a metal. Make sure that disabling the automatic conversion
+    // to zob behaves as expected
+
+    const std::string testfile = (test_samples_path / "metalZobs.mae").string();
```

Comment:
I think that the tests are failing because this is metalZobs.mae and the test file name is metalZob.mae (extra s here).

Reply:
thanks! I fixed that and made the test more precise. Seems like the tests are passing now